CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -37,1032 +37,1033 @@
-        <source>Unfortunately, [The Movie Database](https://www.themoviedb.org/) does not yet support doing a union search for shows. I've asked them to add it; we'll see what happens!</source>
+        <source>Unfortunately, [The Movie Database](https://www.themoviedb.org/) does not yet support doing a union search for shows. I've asked them to add it; we'll see what happens!</source><target>Leider kann man aktuell noch nicht nach Gewerkschaften für TV Sendungen in der [The Movie Database](https://www.themoviedb.org/) suchen. Ich habe bereits nachgefragt. Wir schauen mal, was passiert.</target>
         <note/>
       </trans-unit>
       <trans-unit id="Unknown" xml:space="preserve">
-        <source>Unknown</source>
+        <source>Unknown</source><target>Unbekannt</target>
         <note/>
       </trans-unit>
       <trans-unit id="Upgraded" xml:space="preserve">
-        <source>Upgraded</source>
+        <source>Upgraded</source><target>Erweitert</target>
         <note>Shown in a semi-hidden debugging view</note>
       </trans-unit>
       <trans-unit id="User Interface" xml:space="preserve">
-        <source>User Interface</source>
+        <source>User Interface</source><target>Benutzungsoberfläche</target>
         <note>Header in Persnickety Preferences</note>
       </trans-unit>
       <trans-unit id="Waiting for approval…" xml:space="preserve">
-        <source>Waiting for approval…</source>
+        <source>Waiting for approval…</source><target>Auf Zustimmung warten…</target>
         <note>Shown when waiting for a purchase to be approved (probably by a parent)</note>
       </trans-unit>
       <trans-unit id="Watch in:" xml:space="preserve">
-        <source>Watch in:</source>
+        <source>Watch in:</source><target>Hier anschauen:</target>
         <note>Shown when choosing a country to use for Where to Watch (short version)</note>
       </trans-unit>
       <trans-unit id="Where to Watch" xml:space="preserve">
-        <source>Where to Watch</source>
+        <source>Where to Watch</source><target>Hier anschauen</target>
         <note/>
       </trans-unit>
       <trans-unit id="Which browser to use to open external links." xml:space="preserve">
-        <source>Which browser to use to open external links.</source>
+        <source>Which browser to use to open external links.</source><target>Wähle einen Browser, der zum Öffnen externer Links genutzt werden soll.</target>
         <note/>
       </trans-unit>
       <trans-unit id="Which quick access link is shown." xml:space="preserve">
-        <source>Which quick access link is shown.</source>
+        <source>Which quick access link is shown.</source><target>Folgende Schnellverknüpfung wird angezeigt.</target>
         <note>Shown as a subtitle for a row on Settings</note>
       </trans-unit>
       <trans-unit id="Why not shows?" xml:space="preserve">
-        <source>Why not shows?</source>
+        <source>Why not shows?</source><target>???</target>
```

Comment:
@cliss Could you clarify the meaning/context of the source string?

Reply:
It's here:

![image](https://github.com/cliss/callsheet-localizations/assets/282460/6ada9143-f523-4325-9ab5-0a780b9824ac)

Basically, when you do a cast union search, that only works for movies, and not shows. This button is to get ahead of the inevitable support emails saying HEY HOW COME THIS DOESNT WORK FOR TV‽ 

🙃